factory/processing/processComponents_test.go

```diff
@@ -999,6 +999,7 @@ func getRunTypeComponents(rt runType.RunTypeComponentsHolder) *mainFactoryMocks.
 		ExtraHeaderSigVerifier:              rt.ExtraHeaderSigVerifierHolder(),
 		GenesisBlockFactory:                 rt.GenesisBlockCreatorFactory(),
 		GenesisMetaBlockChecker:             rt.GenesisMetaBlockCheckerCreator(),
+		EpochStartTriggerFactoryField:       rt.EpochStartTriggerFactory(),
```

Comment:
In this file at line 950 you need to add one more test for `nil EpochStartTrigger`

Reply:
Right, good catch, added it